pallets/moonbeam-orbiters/src/lib.rs

```diff
@@ -228,6 +233,13 @@ pub mod pallet {
 			old_orbiter: Option<T::AccountId>,
 			new_orbiter: Option<T::AccountId>,
 		},
+		/// An orbiter has registered
+		OrbiterRegistered {
+			account: T::AccountId,
+			deposit: BalanceOf<T>,
+		},
+		/// An orbiter has unregistered
+		OrbiterUnRegistered { account: T::AccountId },
```

Comment:
```suggestion
		OrbiterUnregistered { account: T::AccountId },
```
Unregistered is a single word so camelcase isn't needed